### DISCUSSION TITLE
creating iModels with .sp file

### DISCUSSION THREAD

**Discussion by ec-dev-1** [iTwin/itwinjs-core#3363] [Q&A] [answered]
What would be the best way to import an .sp file type when creating/registering an iModel?

From the developer dash board, selecting "Local File" did not work and I was given "The file type is not allowed".

I also tried the iTwin Synchronizer and was given the "An iModel connector needed to process these files was not found. Please enable a connector below to proceed" message and no connectors were suggested.

Any help or clarification will be greatly appreciated.

Thank you,

![Screenshot 2022-03-15 142906](https://user-images.githubusercontent.com/86491141/158452831-cf2f291e-9575-43ed-b3bc-e3c6400bac60.png)

![Screenshot 2022-03-15 150612](https://user-images.githubusercontent.com/86491141/158453109-e9da752a-fb2f-481f-9f5a-0dc6efd642a9.png)

**Comment by jffmarker** [comment, marked as answer]
What file format is `.sp`? I don't see it on the official support list: https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/47595/supported-applications-and-file-formats

**Reply by jffmarker** [reply]
@pmconne has advised that this is a SYNCHRO file. That kind of data is handled separately through our SYNCHRO product. [SYNCHRO Control: Creating an iModel with an SP file](https://www.youtube.com/watch?v=XhXOrJxecQ0) may help.
